.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: 🚀 Feature Request
+about: I have a suggestion (and may want to implement it 🙂)!
+
+---
+
+## Feature Request
+
+### Use Case
+**What problem does this feature solve? Please describe.**
+A clear and concise description of what the problem is. Ex. I have an issue when [...]
```

Comment:
Should these be included inside the comment markup ('<!--  What problem does this feature solve? Please describe. --->') like the bug report template?

Reply:
Good catch. Copied this from one of GitHub's examples. Totally missed it. 

Thanks!

Reply:
Fixed.